db/migrate/20170103144759_encrypt_user_credentials.rb

```diff
@@ -0,0 +1,6 @@
+class EncryptUserCredentials < ActiveRecord::Migration[5.0]
+  def change
+      rename_column :user_credentials, :credential_value, :encrypted_credential_value
```

Comment:
Still, we need a way to encrypt / re-encrypt the database (in case of key change), which cannot be a migration. Not sure which is the best way to handle this. Maybe I should just extend `rake` commands and provide something like `bundle exec rake db:encrypt` to encrypt the database using the current `APP_ENCRYPTION_PASSPHRASE` in `.env` file, and `bundle exec rake db:encrypt old_encryption_passphrase` to re-encrypt the whole database (previously encrypted using `old_encryption_passphrase` that will be encrypted with `APP_ENCRYPTION_PASSPHRASE` from `.env` file).

What do you think about it?